src/editor/editor.directive.ts

```diff
@@ -257,40 +259,39 @@ export class FroalaEditorDirective implements ControlValueAccessor {
 
           for (let attr in tags) {
             if (tags.hasOwnProperty(attr) && attr != this.INNER_HTML_ATTR) {
-              this._$element.attr(attr, tags[attr]);
+              this._$element.setAttribute(attr, tags[attr]);
```

Comment:
As `_$element` is a DOM object and not a jQuery object, it should be changed to `_element`.